drivers/epson/projector/esc_vp21.cr

```diff
@@ -0,0 +1,274 @@
+require "digest/md5"
+require "placeos-driver/interface/muteable"
+require "placeos-driver/interface/powerable"
+require "placeos-driver/interface/switchable"
+
+module Epson; end
+
+module Epson::Projector; end
+
+# Documentation: documentation link
+
+class Epson::Projector::EscVp21 < PlaceOS::Driver
```

Comment:
I'm not familiar with the protocol for this device, but unless this is specific to an exact model, can probably make this class make a little more generic.

Reply:
I think ESC/VP21 is just the name of the protocol or maybe group of projectors 
![image](https://user-images.githubusercontent.com/18463731/100390692-32435d00-3029-11eb-98c2-fce1222136cb.png)